mtags-interfaces/src/main/java/scala/meta/pc/PresentationCompiler.java

```diff
@@ -73,6 +74,12 @@
      */
     public abstract CompletableFuture<DefinitionResult> definition(OffsetParams params);
 
+    /**
+     * Decompile .tasty or .class file.
+
+     */
+    public abstract CompletableFuture<Optional<ExecuteCommandParams>> decompile(URI targetUri);
```

Comment:
Maybe we could return string or some data structure that could also be used for json file for other editors.

```suggestion
    public abstract CompletableFuture<Optional<String>> decompile(URI targetUri);
```

This can be easily translated later on to invoke a command.